docs/ignore-oss.md

```diff
@@ -57,23 +57,20 @@ Large files and unknown file extensions are included or excluded through command
 
 This document defines **files, folders and code** as those that are **relevant to a Semgrep scan**. For example, `.jpg` files are not a part of Semgrep's scope and therefore are not part of the scope of this document.
 
-
 ## Customizing ignore behavior
 
 Semgrep provides several methods to customize ignore behavior. Refer to the following table to see which method suits your goal:
 
 | Goal | Method |
 |:---- |:------ |
-| To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` are ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. An empty `.semgrepignore` will make Semgrep scan paths in `.gitignore`. |
-| To ignore files and folders in `.gitignore`. | Add `:include .gitignore` to your `.semgrepignore` file. |
+| To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` are ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. |
```

Comment:
I just realized this isn't a part of the sidebars; we have a redirect in place, so should we just delete this doc?

Reply:
where is it redirecting to? I think this doc is quite useful, but if we have similar content elsewhere...?

Reply:
gotcha - in that doc, do we make clear which options are oss-available? if so then yeah I don't see the point of keeping this

Reply:
The options aren't explicitly flagged as OSS, but if they require SAP, we make that clear in the headers

Reply:
sounds perfect!